### DISCUSSION TITLE
updating bug

### DISCUSSION THREAD

**Discussion by kaawel** [KadeArchive/Kade-Engine#533] [Q&A] [answered]
when i use "lime test windows" it says there's an error and it couldn't find discord_rpc but when i try to install it, it says i already have it and then starts updating, but it doesn't stop updating

![ugh](https://user-images.githubusercontent.com/84515104/119069847-cc83cc80-b9de-11eb-98bf-b859afb6f128.png)

**Comment by Kade-github** [comment, marked as answer]
Refer back to [here](https://kadedev.github.io/Kade-Engine/building)
